### ISSUE TITLE
Submitted reports to maintain data at time of submission

### ISSUE THREAD

**Issue by AureliaKhorsand** [fecgov/fecfile-web-app#1017]
### Business Reason ### 

Submitted reports must maintain data at the time of submission so that it can be a record of what was sent to FEC at the time of submission.

### Acceptance Criteria ### 

**when** the user clicks "Submit" on an open report (Report status: 'In progress')
**Then** all report data will be "frozen" and will not update.

### QA Notes
- [ ] BA/Design Quality Checklist complete

### DEV Notes
- Note that the reports will be able to be amended, or "re-opened" (report status will be re-set to 'In progress')

**Comment by AureliaKhorsand** [comment]
@mjtravers @toddlees @dheitzer @Elaine-Krauss-TCG this ticket is pretty bare, but this is the ticket to "freeze" the report data when a report is submitted. Can you let me know what it's missing, and add dev notes as you see fit?

**Comment by toddlees** [comment]
for the report level,  I think we just need to make sure the summary doesn't get recalculated.  Maybe we add a condition in the report mixin where it 'dirties' a report, to pass if it's closed.

the transactions will be the tricky part i think.  Aggregation and subsequently itemization values are calculated on the fly, so we'll need to store a frozen value for them somewhere.

I'm still not clear on the functional definition of 'submitted' is.  Is it the moment they press submit?  is it only after fec sends a n 'accepted' response?  is it submitted the moment they press submit and 'not submitted' if fec comes back with something other than 'accepted'?  Is 'submitted' the same as 'closed'?

**Comment by AureliaKhorsand** [comment]
Given the decision to allow for data to dynamically change in submitted reports (requirements session 10/19/23), this ticket is no longer needed. Closed as OBE.
